www/content/workstation/upgrade_labs.md

```diff
@@ -0,0 +1,454 @@
+    identifier = "chef_workstation/upgrade_labs.md Upgrade Chef"
+    parent = "chef_workstation"
+    weight = 35
++++
+
+Chef's Upgrade Lab provides an isolated cookbook development environment and in-line support to help you upgrade your system, so you can stop using legacy Chef Infra and start using modern Chef Infra.
+
+The Upgrade Lab provides a report of your existing nodes and cookbooks, so you'll know the scope of the work and you can identify a good place to start. Upgrade Lab works by capturing any node from the production environment and recreating it locally by generating a repository for that node, which provides you with a sandbox to work through upgrading and testing your cookbooks at a safe distance from your production environment.
+
+## About This Guide
+
+This guide covers the straightforward pattern of upgrading from Chef Infra Client 12 to Chef Infra Client 16---including upgrading Chef Infra Server as you go. While we think this is the smoothest path forward, it is not meant to exclude other approaches.
+
+Unless otherwise indicated, you'll run all commands in your local development environment.
+
+## Chef Upgrade Lab Requirements
+
+### System Architecture Requirements
+
+The Chef Lab makes some basic assumptions about your existing system:
+
+* Two Chef Servers
+  -  A Chef Server running some older version
+  -  A newly installed Chef Infra Server running the latest software, set up and configured for knife, but otherwise empty
+* One or more nodes
+  - Running Chef Infra Client 12-15
+  - Minimum 512 MB RAM
+  - [Recommended](https://docs.chef.io/chef_system_requirements/#chef-infra-client) 5GB space
+  - Bootstrapped to the older Chef Server
+  - `sudo` permissions on the node
+  - SSH connectivity between the nodes and the administrator/developer workstation
+* An administrator/developer workstation
+  - 64-bit architecture
+  - Minimum 4 GB RAM
+  - Recommended 10 GB of available disk space for installing Chef Workstation and running the Chef Upgrade Lab
+
+### Software Requirements
+
+* Meet the [platform and system requirements](https://docs.chef.io/workstation/install_workstation/) for Chef Workstation
+* Install or upgrade to the Chef Workstation [latest verision](https://downloads.chef.io/chef-workstation)
+* Any [supported Ruby version](https://www.ruby-lang.org/en/downloads/)
+
+Chef does not prescribe any specific editor. However, the [Chef Infra extension](https://marketplace.visualstudio.com/items?itemName=chef-software.Chef) for [Visual Studio Code](https://code.visualstudio.com/) features several code generators and helpful features, such as running Cookstyle each time you save a recipe.
+
+### Server Backup
+
+We recommend performing a backup before starting any server upgrade process.
+Follow the [Chef Infra Server Backup](https://docs.chef.io/runbook/server_backup_restore/) documentation before starting your Upgrade Lab.
+
+### Chef Infrastructure Requirements
```

Comment:
This could be kind of confusing given we're referring to the product `Chef Infra` elsewhere in this document.

```suggestion
### Infrastructure Requirements
```

maybe?